linux_os/guide/services/ssh/ssh_server/sshd_use_approved_ciphers/rule.yml

```diff
@@ -1,10 +1,23 @@
 documentation_complete: true
 
-prodtype: rhel6,rhel7,ol7,rhv4
+prodtype: rhel6,rhel7,ol7,rhv4,sle12
 
-title: 'Use Only FIPS 140-2 Validated Ciphers'
+title: |-
+{{% if product == "sle12" %}}
+    Use Only DoD-approved encryption Ciphers
+{{% else %}}
+    Use Only FIPS 140-2 Validated Ciphers
+{{% endif %}}
```

Comment:
Nack on this change. FIPS is mandated by Federal Law. This change is not needed or desired.

Reply:
@redhatrises is correct. [NIST provides the language](https://csrc.nist.gov/Projects/cryptographic-module-validation-program) around legal requirements.